src/main/resources/en-CA.yml

```diff
@@ -8,43 +8,6 @@ en-CA:
     internet:
       free_email: [gmail.com, yahoo.ca, hotmail.com]
       domain_suffix: [ca, com, biz, info, name, net, org]
-    phone_number:
-      country_code: ['1']
```

Comment:
I don't necessarily think it's wrong I just want to understand. In some cases country code was left in some it was removed?

Reply:
@kingthorin I left `country_code:` in those "*.yml" files where it was the only value. 
I just was not sure if I can leave these files empty, or should I delete those files. :)
It was easier to leave them untouched, just in case.

Reply:
Okay that's fine.